projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+- **Internal discussion and common sense**: While above sources are already exhaustive, we also used team discussions and our own knowledge.
+
+The requirements have been given a priority based on the [MoSCoW](https://en.wikipedia.org/wiki/MoSCoW_method) scale to allow for tool comparison.
+
+| ID   | Group                  | Requirement                                                  | Priority |
+| ---- | ---------------------- | ------------------------------------------------------------ | -------- |
+| 1... | Functionality          | The tool allows users to perform technical tests on an algorithm or model. | M        |
+|      | Functionality          | The tool allows users to choose which tests to perform.      | M        |
+|      | Functionality          | The tool supports plugin functionality so additional tests can be added easily. | S        |
+|      | Functionality          | The tool can generate a human readable report.               | M        |
+|      | Functionality          | The tool can generate a machine readable report.             | S        |
+|      | Functionality          | The tool allows various formats for reports, like PDF, JSON, MD. | C        |
+|      | Functionality          | The tool allows to create custom reports based on components. | S        |
+|      | Functionality          | It is possible to add custom components for reports.         | S        |
+|      | Functionality          | The tool provides logging.                                   | S        |
+|      | Functionality          | The tool can be used on an isolated system without an internet connection. | S        |
+|      | Functionality          | The tool does not share any data or logging to anyone or anywhere. | C        |
+|      | Functionality          | The tool can be integrated in a CI/CD flow.                  | C        |
+|      | Functionality          | The tool can be offered as a (cloud) service where no local installation is required. | C        |
+|      | Reliability            | The tool can operate consistently and reliably.              | M        |
+|      | Reliability            | The tool can recover automatically from common failures.     | M        |
+|      | Reliability            | The tool recovers from failures quickly and minimizes data loss. | S        |
+|      | Reliability            | The tool handles errors gracefully and informs users of any issues. | S        |
+|      | Reliability            | The tool provides clear error messages and instructions for troubleshooting. | S        |
+|      | Usability              | The tool possess a clean, intuitive, and visually appealing UI that follows industry standards. | S        |
+|      | Usability              | The tool offers keyboard shortcuts for efficient interaction. | C        |
```

Comment:
This one is to much in my oppinion

Reply:
It most definately is a big 'could have' but I'm ok with it being a won't have.. it does however relate to the (only)  Accessibility issue, which requires accessibility features, in web applications (by default) keys like tab, enter etc have a (special) meaning so adding shortcuts to the wishlist is not that far fetched IMO.

Reply:
We could actually instead refer to https://www.digitoegankelijk.nl/toegankelijkheid/en-301-549-en-wcag ?

Reply:
I would agree with @anneschuth here if it would be tool that is client facing, but as for internal usage it wouldn't matter so much (if it is an CLI tool that we incorporate in our own UI).

Reply:
I'm saying that the requirement should maybe focus on a11y in general (and then this would be the reference) instead of just shortcuts. 
And then a second question is how important this for the type of tool we're assessing.

Reply:
I have added the link to WCAG in the accessibility. I still do want to keep this requirement, even as a low 'could'. Usability is often overlooked and UX with keys can be a big plus, without having to be a big draw back if missing.